manifests/s/Saladict/app/3.0.6/Saladict.app.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Saladict.app
+PackageVersion: 3.0.6
+InstallerLocale: zh-CN
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- silentWithProgress
+UpgradeBehavior: install
+Protocols:
+- '0'
+- '3'
+- GPL
+FileExtensions:
+- exe
+ReleaseDate: 2025-01-17
+Installers:
+- Architecture: arm64
+  InstallerUrl: https://github.com/allentown521/saladict/releases/download/3.0.6/Saladict_3.0.6_arm64-setup.exe
+  InstallerSha256: C0A83325146702B70E3A99C263FE3F15A3119AF0FF50A028D87815C103C0641A
+ManifestType: installer
+ManifestVersion: 1.5.0
```

```suggestion
ManifestVersion: 1.9.0
```

Version of manifest are outdated.
清单版本已过时